pyproject.toml

```diff
@@ -2,41 +2,63 @@
 line_length = 130
 target_version = ["py37"]
 
-[tool.isort]
-profile = "black"
-combine_as_imports = true
-line_length = 130
-extra_standard_library = [
+[tool.ruff]
```

Comment:
If we're going to use ruff for linting, why not replace black and use ruff for formatting also?

Reply:
Let's do that in a separate PR I guess, since it may cause a lot of noise.

Reply:
I don't feel strongly about switching to ruff for formatting. We can do it later if someone is inspired.

FWIW, I don't think "typeshed uses it" is a reason to stick with black since if we push something to typeshed it's trivial to run black on it at that time.